### PR TITLE
Smokables can now expose temperature to the atmosphere, causing fires.

### PR DESCRIPTION
**Pull request by gradientvera**
## About the PR 
Cigars and cigarettes cause fires now! Careful where you smoke.

**Screenshots**
Peek 12-01-2022 19-32

**Changelog**

:cl:
- tweak: Lit cigars and cigarettes cause fires now.

### CODE REVIEW

**Comment by MeltedPixel** [comment]
Shouldn't this also be done with lighters and matches?

**Comment by mirrorcult** [comment]
lighters and matches already do this iirc

**Comment by MeltedPixel** [comment]
I guess I don't cause enough mischief.

**Review by ShadowCommander — APPROVED** [review]

File: Content.Server/Nutrition/EntitySystems/SmokingSystem.cs [hunk 1]
```diff
@@ -92,6 +94,13 @@ public override void Update(float frameTime)
                     continue;
                 }
 
+                if (smokable.ExposeTemperature > 0 && smokable.ExposeVolume > 0)
+                {
+                    var transform = Transform(uid);
+                    _atmos.HotspotExpose(transform.Coordinates, smokable.ExposeTemperature, smokable.ExposeVolume, true);
+                }
+
+
```
Comment (ShadowCommander) [inline Content.Server/Nutrition/EntitySystems/SmokingSystem.cs:103]:
```suggestion
```

**Pull request merged**